### ISSUE TITLE
Bool true gets converted to True in TF code

### ISSUE THREAD

**Issue by kzehnter** [DontShaveTheYak/cf2tf#214]
Version 0.6.1

**Expected Behaviour:**
Converting a boolean value should return the words "true" and "false" in lowercase for terraform. 

**Actual Behaviour:**
When converting my resources _true_ gets changed to _True_ even though Terraform needs the lowercase boolean values.

**CF Code:**
``` 
VPC:
    Type: AWS::EC2::VPC
    DeletionPolicy: Retain
    Properties:
      CidrBlock: !Ref VpcCIDR
      EnableDnsSupport: true
      EnableDnsHostnames: true
      InstanceTenancy: default
```

**TF Code:**
```
resource "aws_vpc" "vpc" {
  cidr_block = var.vpc_cidr
  enable_dns_support = True
  enable_dns_hostnames = True
  instance_tenancy = "default"
```

**Comment by kzehnter** [comment]
Apparently this problem is known because I found this:
```
    # todo This needs fixed because python True needs to be terraform true
    return LiteralType(f"!{condition}")
```
in /src/cf2tf/conversion/expressions.py Line 222.
Seems fitting

**Comment by shadycuz** [comment]
Fixed in 0.6.2, feel free to reopen if you still have issues.
